config/oauth.json

```diff
@@ -143,6 +143,13 @@
     "access_url": "https://app.clio.com/oauth/token",
     "oauth": 2
   },
+  "cognito": {
+    "authorize_url": "https://[subdomain]/authorize",
+    "access_url": "https://[subdomain]/token",
+    "profile_url": "https://[subdomain]/userInfo",
```

Comment:
I just set up something similar to get AWS Cognito working with my FeathersJS app, and it looks like `profile_url` should actually be `https://[subdomain]/oauth2/userInfo` based on these docs: https://docs.aws.amazon.com/cognito/latest/developerguide/userinfo-endpoint.html

Reply:
As mentioned in the original PR, if a domain has not been configured the endpoints would be

```
https://cognito-idp.{aws-region}.amazonaws.com/{userPoolId}/authorize
https://cognito-idp.{aws-region}.amazonaws.com/{userPoolId}/token
```

Edit: I have removed the `profile_url` as they would have been in `grant-profile`.

Reply:
To elaborate with example, this will allow both use cases:

```js
// Domain not configured
subdomain = cognito-idp.{aws-region}.amazonaws.com/{userPoolId}
// Domain configured
{domain-prefix}.{aws-region}.amazoncognito.com/oauth2

```

Reply:
I'd say lets go with `https://[subdomain]/oauth2/authorize` and `https://[subdomain]/oauth2/token` by default.

I wasn't even able to get it working with `cognito-idp.[REGION].amazonaws.com/[USER-POOL-ID]` but it works fine with `[DOMAIN].auth.[REGION].amazoncognito.com`

Note that there is an `.auth.` in between the custom domain and the region.

Also note that the user can easily set both the `authorize_url` and the `access_url` and bypass the `subdomain` entirely.

I'll make the necessary adjustments.

Thanks :+1: